addons/point_of_sale/static/src/js/Screens/ProductScreen/ProductsWidget.js

```diff
@@ -63,8 +63,7 @@ class ProductsWidget extends PosComponent {
     get shouldShowButton() {
         return (
             this.productsToDisplay.length === 0 &&
-            this.searchWord &&
-            !this.env.pos.isEveryProductLoaded
+            this.searchWord
```

Comment:
Maybe put it in 1 line since it's not really readable anymore.
return this.productsToDisplay.length === 0 && this.searchWord;

Reply:
Nah it's fine, it's the convention that the JS team is using which is SAM doing for the refactor. So i guess we need to adopt it now